annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Error.java

```diff
@@ -96,11 +101,11 @@ public boolean hasFix() {
   }
 
   public ImmutableSet<Fix> getResolvingFixes() {
-    return ImmutableSet.copyOf(this.resolvingFixes);
+    return this.resolvingFixes;
   }
 
   /**
-   * Checks if error is resolvable with only one fix.
+   * ======= >>>>>>> master Checks if error is resolvable with only one fix.
```

Comment:
And here!

Reply:
ba333098e865557891a1fbf40e49e18f24ae4e75